feat: Added `toggle-mode`, allowing toggle-on and toggle-off (#2555)
zmkfirmware/zmk: docs/docs/keymaps/behaviors/layers.md

&tog LOWER
```

### Configuration

#### Toggle mode

If you wish to ensure that a layer is toggled on or off specifically, rather than switching between the two states, then you can do so with the `toggle-mode` property.
Define a new behavior and assign `"on"` or `"off"` to `toggle-mode`:

```dts
/ {
    behaviors {
        tog_on: toggle_layer_on_only {
            compatible = "zmk,behavior-toggle-layer";
            #binding-cells = <1>;
            display-name = "Toggle Layer On";
            toggle-mode = "on";
        };
    };
};
```

> You can then use `&tog_on` in place of `&tog` whenever you wish to only toggle a layer on, and not toggle it off. The `"off"` version is analogous.

Reviewer: ```suggestion
You can then use `&tog_on` in place of `&tog` whenever you wish to only toggle a layer on, and not toggle it off. An `"off"` version of the behavior can be defined similarly.
```
